addons/website/static/tests/tours/snippet_popup_open_on_top.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module */
+
+import tour from "web_tour.tour";
+import wTourUtils from "website.tour_utils";
+
+function fail (errorMessage) {
+    tour._consume_tour(tour.running_tour, errorMessage);
+}
```

Comment:
Why the need of this? Can't you just do `console.error()`?

Reply:
yes, I copied other tours (I imagined that it is intended to not continue the tour but this is in the last step), I put console.error instead